golos.vesting/golos.vesting.cpp

```diff
@@ -541,5 +545,32 @@ void vesting::timeout_delay_trx() {
     trx.send(_self.value, _self);
 }
 
+void vesting::paydelegator(name voter, asset reward, name delegator, 
+        uint64_t interest_rate, uint8_t payout_strategy) {
```

Comment:
we need `require_auth` here to prevent unauthorised usage.
`_self` auth should be ok